### PR TITLE
Re-format team page to make alumni more visually prominent

### PR DESCRIPTION
**Pull request by adamperer**

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -45,6 +45,34 @@ <h2>Past Members</h2>
     <span class="gray">(formerly a {{ person.role }})</span>
   </li>
   {% endif %} {% endfor %}
-</ul>
+</ul> -->
+
+<div class="flex flex-wrap">
```
Comment (cabreraalex) [inline team.html:50]: maybe add 'mb4' class to push down the 'editing list' section

**Review by cabreraalex — APPROVED** [review]
Looks great! Just a padding thing

File: team.html (hunk 1 above)
Comment (adamperer) [reply team.html:50]: not sure i follow.  where do you want me to add mb-4?

Comment (adamperer) [reply team.html:50]: ah nevermind, i finally parsed what you meant by 'editing list' 🥇 .  see new commit below.

Comment (cabreraalex) [reply team.html:50]: looks good, could also just add it to this div

File: team.html [hunk 2]
```diff
@@ -45,6 +45,35 @@ <h2>Past Members</h2>
...
+            </a>
+          </div>
+      {% endfor %} 
+      </div> 
+      {% endif %}
+      {% endfor %}
+
+</div>
+
+<div class="mb4"></div>
```
Comment (domoritz) [inline team.html:78]: What's this stray div doing? If you want more space, add margin to another container.

File: team.html [hunk 3]
```diff
@@ -36,7 +36,7 @@ <h2>Current Members</h2>
 
 <h2>Past Members</h2>
 
-<ul class="list pl0">
+<!-- <ul class="list pl0">
```
Comment (domoritz) [inline team.html:39]: Let's remove commented out code.

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by domoritz** [comment]
I suggested in the related issue to not group the alumni. This way we can sort them by time and keep the Lay-out a bit more compact. 

I don't have a strong opinion here though. Just a slight preference.

**Pull request merged**